### DISCUSSION TITLE
Document GitLab max role

### DISCUSSION THREAD

**Discussion by WilliamDEdwards** [renovatebot/renovate#23591] [Request Help] [answered]
### How are you running Renovate?

Placeholder value, please select the correct response from the dropdown

### If you're self-hosting Renovate, tell us what version of Renovate you run.

36.25.0

### If you're self-hosting Renovate, select which platform you are using.

gitlab.com

### Was this something which used to work for you, and then stopped?

Placeholder value, please select the correct response from the dropdown

### Describe the problem

AFAICS, the Renovate documentation does not mention anywhere the role that should be granted to the Renovate bot user when adding it as a project member. Testing it leads me to conclude that it requires the 'Developer' role. Otherwise, projects are not autodiscovered.

### Have you created a minimal reproduction repository?

I have explained in the description why a minimal reproduction is impossible

**Comment by viceice** [comment]
feel free to update the docs 🤗

https://github.com/renovatebot/renovate/blob/main/lib/modules/platform/gitlab/index.md

**Reply by WilliamDEdwards** [reply, marked as answer]
Someone else did: https://github.com/renovatebot/renovate/commit/739b6e4d726940748af4a084fc13b341f09c445c
